packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts

```diff
@@ -122,14 +120,15 @@ export const getStyles = (props: INavStyleProps): INavStyles => {
       !isDisabled && {
         selectors: {
           '.ms-Nav-compositeLink:hover &': {
-            backgroundColor: palette.neutralLighterAlt,
+            backgroundColor: semanticColors.bodyBackgroundHovered,
             color: semanticColors.bodyText
           }
         }
       },
       isSelected && {
-        color: palette.themePrimary,
-        backgroundColor: palette.neutralLighter,
+        color: semanticColors.bodyTextChecked,
+        fontWeight: FontWeights.semibold, // todo: get from theme
```

Comment:
just curious on the comment. Do we plan to have a theme property for font weights? My thoughts were that we should probably have font sizes so that when building the fonts we would pull from them (similar to how we do with semantic slots) but for font weights not sure actually, unless we are planning to have semantic slots for them like `fontWeights.heading`, `fontWeights.paragraph` and so on. The reason I ask is because when I did my PR #10027 I left all the `FontWeights` imports from the styling package as I didn’t know how to deal with them.

Reply:
Right, we haven't decided on how to expose this yet. Fonts are complicated, so we basically punted on it for now. For example, if we switch to having two different fonts for headings and body, how do we expose that? Lots of unanswered questions and unclear use scenarios, so just kicking the can down the road in the meantime.